packages/views.client.unstable/src/host.ts

```diff
@@ -41,7 +41,7 @@ export class FoundryHostEventTarget<
       | null,
     options?: AddEventListenerOptions | boolean,
   ): void {
-    super.addEventListener(type, callback, options);
+    super.addEventListener(type, callback as EventListener, options);
```

Comment:
i had to drop the `extends` else the event is `Event | CustomEvent`, which is annoying for consumers to deal with when the type of the event is typesafe